src/react/components/pages/editorPage/canvas.tsx

```diff
@@ -75,12 +74,21 @@ export default class Canvas extends React.Component<ICanvasProps, ICanvasState>
             this.clearAllRegions();
             this.setState({
                 currentAsset: this.props.selectedAsset,
-                selectedRegions: [],
             });
         }
 
         if (this.props.selectionMode !== prevProps.selectionMode) {
-            this.editor.AS.setSelectionMode(this.props.selectionMode, null);
+            let  options = null;
+            if (this.props.selectionMode === SelectionMode.COPYRECT) {
+                const selectedRegion = this.getSelectedRegions(); // this.state.selectedRegions[0];
+                if (selectedRegion) {
+                    options = {template: new Rect(selectedRegion[0].boundingBox.width,
+                                                selectedRegion[0].boundingBox.height)};
```

Comment:
It should be just `template`:
{ 
  mode: SelectionMode.COPYRECT,
  template: new Rect(...),
}